crates/common/Cargo.toml

```diff
@@ -18,4 +18,4 @@ default = []
 error_as_host = []
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-wasmer = "=4.2.8"
+wasmer = "=4.3.0"
```

Comment:
Unrelated, thought I'd bump to see if it fixed anything but it didn't and now I don't see a reason to drop this change

Reply:
Never mind, can't use the latest because of a yanked libc